recipe/meta.yaml

```diff
@@ -5,32 +5,36 @@
 {% set hash_value = "4765c5f49eb4c8cfbfe87609c8797313c3b8259728ae5f7b224a4827d9640372" %}
 
 package:
-  name: '{{ name|lower }}'
-  version: '{{ version }}'
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  fn: {{ name }}-{{ version }}.{{ file_ext }}
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
-  '{{ hash_type }}': '{{ hash_value }}'
+  {{ hash_type }} : {{ hash_value }}
 
 build:
+  skip: True   # [win]
```

Comment:
These two will skip the entire platform.

Reply:
i know but those tests have already passed. will remove the flag after i figure out what does the trick on linux...

Reply:
ah! oki! 👍

Reply:
do you know why the appveyor stuff sometimes fails because of an SSH certificate?

Reply:
no clue, looks like a problem on their end...